### PR TITLE
Fixed changelog

### PR DESCRIPTION
**Pull request by ferrandinand**
Changelog had a mistake.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.4.0] 2019-09-24
+- Add message to known errors link if pushing to catalog or collection fails.
+
+## [0.4.0] 2019-09-17
```
Comment (TheoBrigitte) [inline CHANGELOG.md]: We are missing the link for this at the bottom of this file, their is a duplicate [0.3.0].

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,12 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.4.0] 2019-09-24
+- Add message to known errors link if pushing to catalog or collection fails.
```
Comment (TheoBrigitte) [inline CHANGELOG.md]: Missing the Added section before this.

File: CHANGELOG.md (hunk 1 above)
Comment (ferrandinand) [reply CHANGELOG.md]: Solved 0.3.0 error link.
Not sure why we are missing a lot of information about the unreleased version in comparison with 0.4.0 but the link should be still the same isn´t it?

[Unreleased]: https://github.com/giantswarm/architect-orb/compare/v0.4.0...HEAD

Comment (TheoBrigitte) [reply CHANGELOG.md]: I don't either, but this look good now. Thanks for catching this.

**Pull request merged**